program/find-surface-area-of-a-cylinder/FindSurfaceAreaOfACylinder.js

```diff
@@ -0,0 +1,8 @@
+function surfaceArea(radius, height) {
+	//formular for surface area
+	area = 2 * Math.PI * radius * (radius + height);
+	return area;
```

Comment:
Why area is declared in global scope ?
Can you make it within functions cope

Reply:
@anandfresh 

Thank you so much for the feedback! I am still new to JavaScript and appreciate all the feedback I can get to help me grow and become better at programming.

Let me try to fix this.

Reply:
> Why area is declared in global scope ? Can you make it within functions cope

@anandfresh Please, can you kindly review my correction and see if it's up to standard. Thank you.